projects/plugins/boost/app/assets/src/js/features/critical-css/lib/generate-critical-css.ts

```diff
@@ -187,13 +185,22 @@ async function generateForKeys(
 	callbacks: ProviderCallbacks,
 	signal: AbortSignal
 ): Promise< void > {
+	const CriticalCSSGenerator = await import(
+		/* webpackChunkName: "jetpack-critical-css-gen" */ '@automattic/jetpack-critical-css-gen'
+	);
```

Comment:
Can we extract the import into it's own function as it's happening in two places?

Reply:
Nice catch! Told you you'd find something 😅 

Updated in 71860ddb4d01eb51514775cf7ec4ac196961ef11.